examples/fluent-bit/send-fb-internal-metrics-to-cw/README.md

```diff
@@ -0,0 +1,77 @@
+# Send Fluent Bit internal Metrics to CloudWatch
+
+This example shows you how to monitor Fluent Bit, by having it upload its own metrics to CloudWatch. 
+
+### How it works
+
+Fluent Bit scrapes its own [prometheus endpoint](https://docs.fluentbit.io/manual/administration/monitoring) and then parses the logs to a JSON that looks like this after FireLens adds ECS Metadata:
+
+```
+{
+    "metric": "fluentbit_output_retries_total",
+    "plugin": "cloudwatch_logs.0",
+    "value": "0",
+    "time": "1649288960583",
+    "ecs_cluster": "firelens-testing",
+    "ecs_task_arn": "arn:aws:ecs:ap-south-1:144718711470:task/firelens-testing/f2ad7dba413f45ddb4d92f7853b78469",
+    "ecs_task_definition": "fluentbit-metrics-to-cw-firelens-example:4",
+    "hostname": "ip-10-192-21-104.ap-south-1.compute.internal"
+}
+```
+
+This example shows a hostname field added, which is the ENI IP of the Fargate task. The config to add hostname is optional and is commented out in the example `extra.conf` in this repo. 
+
+In the CW Log Group for these logs, you can then create a [CloudWatch Metric Filter](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html) to create metrics from the logs. Metric dimensions are customizeable and can be any value in the log JSON. 
+
+Each Fluent Bit instance will upload these logs to a log stream named by its hostname, this should ensure unique log streams are used for each Fluent Bit instance. 
+
+Below is a screenshot that shows how the metrics will look in CloudWatch:
+
+![Screenshot of FB Metrics in CW](screenshot.png?raw=true "Fluent Bit Metrics in CloudWatch")
+
+### Tutorial
+
+### 1. Custom Fluent Bit Configuration
+
+This example contains the following:
+1. Custom Fluent Bit configuration that:
+     * Enables the [Fluent Bit monitoring endpoint](https://docs.fluentbit.io/manual/administration/monitoring)
```

Comment:
nit: to keep same format as below, use `enables`

Reply:
addressed